src/govuk/components/components.template.test.js

```diff
@@ -52,7 +52,7 @@ describe('Components', () => {
 
           // Allow for multiple buttons in the same form to have the same name
           // (as in the cookie banner examples)
-          'form-dup-name': 'off',
+          'form-dup-name': ['error', { shared: ['radio', 'checkbox', 'submit'] }],
```

Comment:
This is fab, glad we can turn it back on 😊 

Always going to be edge cases like "standalone" checkboxes (duplicates = bad) but better than `off`!

<img width="402" alt="Standalone checkboxes" src="https://user-images.githubusercontent.com/415517/211529787-33ffd15f-77f4-41e1-abac-d00fdcc4891f.png">